plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererFileContextProvider.kt

```diff
@@ -219,55 +226,115 @@ class DefaultCodeWhispererFileContextProvider(private val project: Project) : Fi
         val query = generateQuery(targetContext)
 
         val contexts = withContext(coroutineContext) {
-            val projectContextDeferred1 = if (CodeWhispererFeatureConfigService.getInstance().getInlineCompletion()) {
-                async {
-                    val t0 = System.currentTimeMillis()
-                    val r = fetchProjectContext(query, psiFile, targetContext)
-                    val t1 = System.currentTimeMillis()
-                    LOG.debug {
-                        buildString {
-                            append("time elapse for fetching project context=${t1 - t0}ms; ")
-                            append("numberOfChunks=${r.contents.size}; ")
-                            append("totalLength=${r.contentLength}")
-                        }
+            val projectContextDeferred1 = async {
+                val t0 = System.currentTimeMillis()
+                val codemapContext = fetchProjectContext(query, psiFile, targetContext)
+                val t1 = System.currentTimeMillis()
```

Comment:
can we finally use https://kotlinlang.org/api/core/1.9/kotlin-stdlib/kotlin.time/measure-timed-value.html
?

Reply:
i shouldd